### PR TITLE
Tandem Video

### PR DESCRIPTION
**Pull request by kemiller**
## What does this PR do?

This adds support for hosting meetings on Tandem.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

I tested connecting the integration, authorizing a user, and scheduling meetings against our internal servers. 

This code should be ready for soft launch.  We'll want to do a round of testing once you are ready to host.  

To test, you'll just need a free account on Tandem, then go to one of your event types and add Tandem Video and schedule en event.  It should add a tandem link, and add events visible in Tandem, and it should also update correctly when you reschedule or cancel.  

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code and corrected any misspellings
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by PeerRich** [comment]
awesome! can't wait to go live with this App! 🎉

File: lib/integrations/getIntegrations.ts [hunk 1]
```diff
@@ -72,6 +73,14 @@ export const ALL_INTEGRATIONS = [
     description: "Video Conferencing",
     variant: "conferencing",
   },
+  {
+    installed: true,
```
Comment (zomars) [inline lib/integrations/getIntegrations.ts:77]: We should check for the env variables here to consider it installed. We should think of self-hosted instances that don't want/need every integration.

File: pages/api/integrations/tandemvideo/callback.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getSession } from "@lib/auth";
+
+import prisma from "../../../../lib/prisma";
```
Comment (zomars) [inline pages/api/integrations/tandemvideo/callback.ts:5]:
```suggestion
import prisma from "@lib/prisma";
```

File: prisma/zod/attendee.ts [hunk 3]
```diff
@@ -1,7 +1,6 @@
-import * as z from "zod";
```
Comment (zomars) [inline prisma/zod/attendee.ts:1]: It seems that the zod schemas got committed without prettier formatting. Try running `yarn generate-schemas` locally to fix this.

**Review by zomars — COMMENTED** [review]
Codewise looks good. Just left some comments there

File: lib/integrations/getIntegrations.ts (hunk 1 above)
Comment (kemiller) [reply lib/integrations/getIntegrations.ts:77]: Ah, thanks.  I hardcoded it while developing & forgot to circle back.

**Comment by zomars** [comment]
@kemiller Can you please rebase with the main branch? Other than that is all good

**Review by PeerRich — APPROVED** [review]

**Review by zomars — CHANGES_REQUESTED** [review]
Lets remove this file please:

`tsconfig.tsbuildinfo`

**Comment by kemiller** [comment]
> Lets remove this file please:
> 
> `tsconfig.tsbuildinfo`

That's not a file I added.  It came in with a merge from master.  I can rebase onto master to clean up the history, which should take care of it, if you don't mind rebases.

**Review by alishaz-polymath — DISMISSED** [review]
Looks good except what @zomars mentioned already, please remove `tsconfig.tsbuildinfo` file

**Comment by alishaz-polymath** [comment]
> > Lets remove this file please:
> > `tsconfig.tsbuildinfo`
> 
> That's not a file I added. It came in with a merge from master. I can rebase onto master to clean up the history, which should take care of it, if you don't mind rebases.

You can simply just remove the file and push the changes, that should be fine :)

**Review by pumfleet — COMMENTED** [review]
This looks good to me too. Once the tsconfig is resolved, happy to merge

**Comment by PeerRich** [comment]
> This looks good to me too. Once the tsconfig is resolved, happy to merge

we can also merge this and I remove the tsconfig in a follow-up PR, no?

**Comment by zomars** [comment]
> It came in with a merge from master.

If it came from master there wouldn't be a file difference. 

Let's remove the file and add it to `.gitignore` as well to prevent polluting the main branch

**Comment by kemiller** [comment]
> > It came in with a merge from master.
> 
> If it came from master there wouldn't be a file difference.
> 
> Let's remove the file and add it to `.gitignore` as well to prevent polluting the main branch

 It was added here:  [1d1087489017141d239af3e99900b7c6e9e9e634](https://github.com/calcom/cal.com/commit/1d1087489017141d239af3e99900b7c6e9e9e634).  Not sure why it showed up in the diff before, and interestingly it was already in the gitignore, so not sure what happened.  At any rate, I removed it in my branch so we should be GTG.

**Comment by PeerRich** [comment]
merged! epic 👏👏

**Pull request merged**